parm/config/config.ufs

```diff
@@ -354,6 +354,9 @@ if [[ "${skip_ww3}" == "false" ]]; then
     "mx025")
       ntasks_ww3=80
       ;;
+    "gefsv13_025")
+      ntasks_ww3=262
```

Comment:
Probably this one too.

Reply:
I think it'd be nice to have this setting here even though it's "for GEFS".    The resources are probably to get it to run at production speed versus "just finish in some reasonable sort of time frame"//be small enough resources for quick testing.

Reply:
Okay, keeping the setting doesn't really hurt anything.

Reply:
This needs an explanation. 
The case statement is for grid resolution or type.
If someone wants to hard-wire something that just works, this is not the place.

Reply:
This is for a specific grid.  I'm a little confused by the comment of hard-coding or how what @NeilBarton-NOAA added here is any different than any of the above cases.

Reply:
The comment is about naming of the specific grid.  The other names in the case block are based on basin, resolution, and not an application.